__tests__/eslintPlugin/noAnimatedStyleToNonAnimatedComponent.test.tsx

```diff
@@ -0,0 +1,121 @@
+import { RuleTester } from '@typescript-eslint/rule-tester';
+import noAnimatedStyleToNonAnimatedComponent from '../../eslintPlugin/src/noAnimatedStyleToNonAnimatedComponent';
```

Comment:
I don't quite like the fact that we are not testing the bundled output, but source code here instead. We should do a require here for `../eslintPlugin/`.